tests/plugins/colordialog/colordialog.js

```diff
@@ -66,8 +66,49 @@
 		// (#2639)
 		'test background color of mixed selection': function() {
 			openDialogManually( this.editor, '', '<h1>[<span style="background-color:#0000ff">Foo</span>bar]</h1>', 'BGColor' );
-		}
+		},
+
+		// (#4351)
+		'test converting RGBA to hex color': function() {
+			// IE8 doesn't set unsupported values.
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) {
+				assert.ignore();
+			}
```

Comment:
Bender has a rarely used feature of [`_should: { ignore {} }`](https://github.com/cksource/ckeditor4-security-tests/blob/master/plugins/colorbutton/colorvalidation.js#L12). It is not super-useful because when you look at test itself you don't know if it's ignored or not, but since we have a few tests ignored for the same reason here, maybe we could use it here and only describe the reason one?

Reply:
TBH it's such an obscure feature that I would stick with ignoring in tests themselves. It conveys more information to the developer.

Reply:
OTOH ignores in every test add so many clutter that I eventually rewrote it to `_should.ignore`.